pulser/devices/_device_datacls.py

```diff
@@ -47,7 +51,8 @@ class Device:
     max_radial_distance: int
     min_atom_distance: int
     _channels: tuple[tuple[str, Channel], ...]
-    interaction_coeff: float = 5008713.
+    interaction_coeff_ising: float = 5008713.
```

Comment:
Why ? What does that change for the rest of the codebase ?

Reply:
Nothing on this codebase, but there are custom devices outside of it who specify a different value for this coefficient, so this would be a breaking change.